db/migrate/.rubocop.yml

```diff
@@ -7,3 +7,5 @@ Naming/VariableNumber:
 Rails/CreateTableWithTimestamps:
   Include:
     - '*.rb'
+  Exclude:
+    - 20241104082851_create_annual_report_statuses_per_account_counts.rb
```

Comment:
What is the reason for this exclusion here?

As an alternative, would it be possible to disable the offending cop in the migration itself and maybe add a comment with the reason?

Reply:
Reason for exclusion is that the timestamps would have no use in this table. But yes I can move it to the file, done.

Reply:
Thanks, that makes it a lot clearer imo. No need even for an additional comment.